examples/nextjs/src/app/utils/index.js

```diff
@@ -0,0 +1,24 @@
+import { permanentRedirect, redirect } from "next/navigation";
```

Comment:
Will this only work with Next?

Reply:
@oidacra Only in NextJS for now since it has backend capabilities, so there is a specific way to handle redirects